<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMain</class>
 <widget class="QDialog" name="frmMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetTitle" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labIco">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labTitle">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widgetTop" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="btnMain">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Shell</string>
           </property>
           <property name="icon">
            <iconset resource="../other/main.qrc">
             <normaloff>:/image/main_main.png</normaloff>:/image/main_main.png</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="MemTabletoolButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>MemTable</string>
           </property>
           <property name="icon">
            <iconset resource="../other/main.qrc">
             <normaloff>:/image/main_config.png</normaloff>:/image/main_config.png</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnData">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Process</string>
           </property>
           <property name="icon">
            <iconset resource="../other/main.qrc">
             <normaloff>:/image/main_data.png</normaloff>:/image/main_data.png</iconset>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="DeskTabletoolButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>DeskTable</string>
           </property>
           <property name="icon">
            <iconset resource="../other/main.qrc">
             <normaloff>:/image/main_person.png</normaloff>:/image/main_person.png</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnExit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Device</string>
           </property>
           <property name="icon">
            <iconset resource="../other/main.qrc">
             <normaloff>:/image/main_exit.png</normaloff>:/image/main_exit.png</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widgetMenu" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="1" column="1" colspan="3">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="btnMenu_Min">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="toolTip">
            <string>最小化</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="btnMenu_Close">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="toolTip">
            <string>关闭</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="btnMenu_Max">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>6</number>
     </property>
     <widget class="QWidget" name="shell_page">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextBrowser" name="echo_textBrowser"/>
       </item>
       <item>
        <widget class="QLabel" name="pwd_label">
         <property name="text">
          <string>PWD：/root</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="cmd_lineEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mem_page">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTextBrowser" name="mem_textBrowser"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="memTable_page">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTableWidget" name="men_tableWidget"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="proc_page">
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="2">
        <layout class="QHBoxLayout" name="execTime">
         <item>
          <widget class="QLabel" name="execTime_label">
           <property name="text">
            <string>执行时间：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="execTime_value">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="startTIME">
         <item>
          <widget class="QLabel" name="startTime_label">
           <property name="text">
            <string>开始时间：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="startTime_value">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="block_label">
         <property name="text">
          <string>阻塞</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="ready_label">
         <property name="text">
          <string>就绪</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="PPID">
         <item>
          <widget class="QLabel" name="PPID_label">
           <property name="text">
            <string>当前父进程号：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="PPID_value">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="PID">
         <item>
          <widget class="QLabel" name="PID_label">
           <property name="text">
            <string>当前进程号：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="PID_value">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="QTableWidget" name="ready_table">
         <column>
          <property name="text">
           <string>进程号</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>父进程号</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>文件名</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>指令</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>开始时间</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>执行时间</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>优先级</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>大小</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="curTime">
         <item>
          <widget class="QLabel" name="curTime_label">
           <property name="text">
            <string>当前时间：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="curTime_value">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="7" column="0" colspan="3">
        <widget class="QTableWidget" name="block_table">
         <column>
          <property name="text">
           <string>进程号</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>父进程号</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>文件名</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>指令</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>开始时间</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>执行时间</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>优先级</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>大小</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="IR">
         <item>
          <widget class="QLabel" name="IR_label">
           <property name="text">
            <string>指令寄存器：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="IR_value">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="2">
        <layout class="QHBoxLayout" name="Pri">
         <item>
          <widget class="QLabel" name="Pri_label">
           <property name="text">
            <string>优先级：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Pri_value">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="deskTable_page">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="totTab_label">
         <property name="text">
          <string>磁盘空间：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="usedTab_label">
         <property name="text">
          <string>已使用：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="freeTab_label">
         <property name="text">
          <string>剩余：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QTableWidget" name="tableWidget"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="desk_page">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="tot_label">
         <property name="text">
          <string>磁盘空间：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="used_label">
         <property name="text">
          <string>已使用空间：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="deskfree_label">
         <property name="text">
          <string>空闲空间：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QTextBrowser" name="desk_textBrowser"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="device_page">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="device__label">
         <property name="text">
          <string>I/O</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTextBrowser" name="device_textBrowser"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../other/main.qrc"/>
 </resources>
 <connections/>
</ui>
